armtemplates/template.bicep

```
@@ -0,0 +1,280 @@
+        
+        properties: {
+          protocol: 'TCP'
+          sourcePortRange: '*'
+          destinationPortRange: '22'
+          sourceAddressPrefix: '*'
+          destinationAddressPrefix: '*'
+          access: 'Allow'
+          priority: 300
+          direction: 'Inbound'
+          sourcePortRanges: []
+          destinationPortRanges: []
+          sourceAddressPrefixes: []
+          destinationAddressPrefixes: []
+        }
+      }
+      {
+        name: 'HTTP'
+        //id: networkSecurityGroups_iagora_nsg_name_HTTP.id
+        //type: 'Microsoft.Network/networkSecurityGroups/securityRules'
+        properties: {
+          protocol: 'TCP'
+          sourcePortRange: '*'
+          destinationPortRange: '80'
+          sourceAddressPrefix: '*'
+          destinationAddressPrefix: '*'
+          access: 'Allow'
+          priority: 320
+          direction: 'Inbound'
+          sourcePortRanges: []
+          destinationPortRanges: []
+          sourceAddressPrefixes: []
+          destinationAddressPrefixes: []
+        }
+      }
+      {
+        name: 'HTTPS'
+       // id: networkSecurityGroups_iagora_nsg_name_HTTPS.id
+        //type: 'Microsoft.Network/networkSecurityGroups/securityRules'
+        properties: {
+          protocol: 'TCP'
+          sourcePortRange: '*'
+          destinationPortRange: '443'
+          sourceAddressPrefix: '*'
+          destinationAddressPrefix: '*'
+          access: 'Allow'
+          priority: 340
+          direction: 'Inbound'
+          sourcePortRanges: []
+          destinationPortRanges: []
+          sourceAddressPrefixes: []
+          destinationAddressPrefixes: []
+        }
+      }
+    ]
+  }
+}
+
+
+//PULIC IP ADDRESS RESSOURC
+
+resource publicIPAddresses_iagora_ip_name_resource 'Microsoft.Network/publicIPAddresses@2023-11-01' = {
+  name: publicIPAddresses_iagora_ip_name
+  location: 'francecentral'
+  sku: {
+    name: 'Standard'
+    tier: 'Regional'
+  }
+  zones: [
+    '1'
+  ]
+  properties: {
+    ipAddress: '4.233.216.162'
+    publicIPAddressVersion: 'IPv4'
+    publicIPAllocationMethod: 'Static'
+    idleTimeoutInMinutes: 4
+    ipTags: []
+  }
+}
+
+// VIRTUAL NETWORK RESSOURC
+
+resource virtualNetworks_iagora_vnet_name_resource 'Microsoft.Network/virtualNetworks@2023-11-01' = {
+  name: virtualNetworks_iagora_vnet_name
+  location: 'francecentral'
+  properties: {
+    addressSpace: {
+      addressPrefixes: [
+        '10.0.0.0/16'
+      ]
+    }
+    subnets: [
+      {
+        name: 'default'
+        //id: virtualNetworks_iagora_vnet_name_default.id
+        properties: {
+          addressPrefix: '10.0.0.0/24'
+          delegations: []
+          privateEndpointNetworkPolicies: 'Disabled'
+          privateLinkServiceNetworkPolicies: 'Enabled'
+        }
+        //type: 'Microsoft.Network/virtualNetworks/subnets'
+      }
+    ]
+    virtualNetworkPeerings: []
+    enableDdosProtection: false
+  }
+}
+
+// VIRTUAL MACHINE
```

```suggestion
// VIRTUAL MACHINE RESSOURCE
```
Nit Pick: for consistency, we could either always add RESSOURCE in the title or always omit it) please choose one but not both)